native-base-theme/variables/commonColor.js

```diff
@@ -162,7 +162,8 @@ export default {
   iosStatusbar: "light-content",
   toolbarDefaultBorder: "#2874F0",
   get statusBarColor() {
-    return color(this.toolbarDefaultBg).darken(0.2).hex();
+    return '#fff';
+    // return color(this.toolbarDefaultBg).darken(0.2).hex();
```

Comment:
i commented all of these out in native-base variables, feel free to add back and debug, i just wanted to get it working